tests/l2/dsa/dsa_build.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: intel-dsa-workload
+  namespace: intel-dsa
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: intel-dsa-workload
+  namespace: intel-dsa
+spec:
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  runPolicy: "Serial"
+  source:
+    type: Dockerfile
+    dockerfile: |
+        
```

Comment:
if possible, I suggest to maintain the docker file in a separate file. And it is easy to maintain.

Reply:
lets not do that right now. all other tests also needs to be changed.